tests/functional/spec/relationships-unrelated.spec.js

```diff
@@ -0,0 +1,92 @@
+      browser.openQuestionnaire(schema);
+    });
+
+    describe("And I add six people", () => {
+      before("add people", () => {
+        addPerson("Andrew", "Austin");
+        addPerson("Betty", "Burns");
+        addPerson("Carla", "Clark");
+        addPerson("Daniel", "Davis");
+        addPerson("Eve", "Elliot");
+        $(ListCollectorPage.no()).click();
+        $(ListCollectorPage.submit()).click();
+      });
+
+      it("When I answer 'Unrelated' twice, Then I will be asked if anyone else is related with a list of the remaining people", () => {
+        $(RelationshipsPage.unrelated()).click();
+        $(RelationshipsPage.submit()).click();
+        $(RelationshipsPage.unrelated()).click();
+        $(RelationshipsPage.submit()).click();
+        expect($(RelatedToAnyoneElsePage.questionText()).getText()).to.contain("Are any of these people related to you?");
+        expect($(RelatedToAnyoneElsePage.listLabel(1)).getText()).to.equal("Daniel Davis");
+        expect($(RelatedToAnyoneElsePage.listLabel(2)).getText()).to.equal("Eve Elliot");
+      });
+
+      it("When I click previous, Then I will go back to the previous relationship", () => {
+        $(RelatedToAnyoneElsePage.previous()).click();
+        expect($(RelationshipsPage.questionText()).getText()).to.contain("Carla Clark is unrelated to Andrew Austin");
+      });
+
+      it("When I return to the 'related to anyone else' question and select 'Yes', Then I will be taken to the next relationship for the first person", () => {
+        $(RelationshipsPage.submit()).click();
+        $(RelatedToAnyoneElsePage.yes()).click();
+        $(RelatedToAnyoneElsePage.submit()).click();
+        expect($(RelationshipsPage.questionText()).getText()).to.contain("Thinking of Andrew Austin, Daniel Davis is their");
+      });
+
+      it("When I click previous, Then I will go back to the 'related to anyone else' question", () => {
+        $(RelationshipsPage.previous()).click();
+        expect($(RelatedToAnyoneElsePage.questionText()).getText()).to.contain("Are any of these people related to you?");
+        expect($(RelatedToAnyoneElsePage.yes()).isSelected()).to.be.true;
+      });
+
+      it("When I select 'No' to the 'related to anyone else' question, Then I will be taken to the first relationship for the second person", () => {
+        $(RelatedToAnyoneElsePage.no()).click();
+        $(RelatedToAnyoneElsePage.submit()).click();
+        expect($(RelationshipsPage.questionText()).getText()).to.contain("Thinking of Betty Burns, Carla Clark is their");
+      });
+
+      it("When I click previous, Then I will go back to the 'related to anyone else' question for the first person", () => {
+        $(RelationshipsPage.previous()).click();
+        expect($(RelatedToAnyoneElsePage.questionText()).getText()).to.contain("Are any of these people related to you?");
+        expect($(RelatedToAnyoneElsePage.no()).isSelected()).to.be.true;
+      });
```

Comment:
I know there is now an integration test for it, but we should assert that only the relevant people are listed after clicking Previous from No.

Reply:
Done.